openmc/mesh.py

```diff
@@ -210,3 +227,90 @@ def get_mesh_xml(self):
             subelement.text = ' '.join(map(str, self._width))
 
         return element
+
+    def build_cells(self, bc=['reflective'] * 6):
+        """Generates a list of cells which mimic the mesh geometry.
```

Comment:
"mimicking the mesh geometry" -> perhaps clarify this to mean that you build separate parallelepipeds for each individual Cartesian mesh cell.